app/src/main/res/values-vi/array.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="off_type">
+        <item>Nghỉ có lương - Công ty trả</item>
+        <item>Nghỉ có lương - Bảo hiểm trả</item>
+        <item>Nghỉ không lương</item>
+    </string-array>
+
+    <string-array name="status">
```

Comment:
FATAL: [ExtraTranslation] The resource string "`status`" has been marked as `translatable="false"`
 If a string appears in a specific language translation file, but there is no corresponding string in the default locale, then this string is probably unused. (It's technically possible that your application is only intended to run in a specific locale, but it's still a good idea to provide a fallback.).

Note that these strings can lead to crashes if the string is looked up on any locale not providing a translation, so it's important to clean them up.